inc/zoo/AnyCallable.h

```diff
@@ -29,6 +29,11 @@ struct AnyCallable;
 /// as they would any type erased value
 template<typename TypeErasureProvider, typename R, typename... Args>
 struct AnyCallable<TypeErasureProvider, R(Args...)>: TypeErasureProvider {
+    static constexpr auto emptyInvoker_ =
```

Comment:
I see the need for `emtpyInvoker_` since it is the basis for `empty` and the rest.  At this point there is no reason to keep it as a lambda instead of a function